modules/primer-marketing-utilities/docs/margin.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Margin
+status: New release
+status_issue: https://github.com/github/design-systems/issues/378
+---
+
+Margin spacing utilities can be applied to elements to achieve bigger y-directional spaces beyond the `primer-core` spacing scale.
+See [primer-marketing-support](https://github.com/primer/primer-css/tree/master/modules/primer-marketing-support) for the extended marketing spacing scale.
+
+## Y-Directional margin spacing
+
+Use y-directional utilities to apply margin to the top, bottom or Y axis of an element. Directional utilities can change or override default margins, and can be used with a spacing scale of 7-11.
```

Comment:
I notice you're using "y-directional" quite a lot through the docs. It doesn't really roll off the tongue very well and I don't think it's a commonly used term 😬. In this instance I think you can get away from using by changing the sentence to the following:

> Use marketing margin utilities to apply margin to top, bottom, or y-axis of an element.

I know it says "margin" twice but kinda difficult to get away from that here.

Reply:
Haha great point @broccolini ! I think i pulled it from our usage of "directional margins" and ran with it... agreed, axis is a much better term. I'll update it!